src/AasxServerDB/Query.cs

```diff
@@ -461,9 +489,9 @@ private List<SMEResult> GetSMEResult(List<SMEWithValue> smeList)
             return smeList.ConvertAll(
                 sme =>
                 {
-                    var identifier = (sme != null && sme.sm.Identifier != null) ? sme.sm.Identifier : "";
-                    var path = sme.sme.IdShort;
-                    var pId = sme.sme.ParentSMEId;
+                    string identifier = (sme != null && sme.sm.Identifier != null) ? sme.sm.Identifier : "";

```

Comment:
I would prefer 'var' as this is the csharp best practices

Reply:
Changed